documentai/snippets/handle_response_sample.py

```diff
@@ -463,13 +464,17 @@ def process_document_layout_sample(
     processor_version: str,
     file_path: str,
     mime_type: str,
+    enable_image_annotation: bool = False, # Enable image annotation
+    enable_table_annotation: bool = False, # Enable table annotation
 ) -> documentai.Document:
     process_options = documentai.ProcessOptions(
         layout_config=documentai.ProcessOptions.LayoutConfig(
             chunking_config=documentai.ProcessOptions.LayoutConfig.ChunkingConfig(
                 chunk_size=1000,
                 include_ancestor_headings=True,
-            )
+            ),
+            enable_image_annotation=enable_image_annotation,
+            enable_table_annotation=enable_table_annotation,
```

Comment:
I would recommend just hardcoding both of these as `True` and have a comment marking as optional.
```suggestion
            https://cloud.google.com/document-ai/docs/layout-parse-chunk#layout_parser_features
            enable_image_annotation=True, # Optional
            enable_table_annotation=True, # Optional
```